benchmarkapps/BasicApi/Controllers/PetController.cs

```diff
@@ -25,6 +25,9 @@ public PetController(BasicApiContext dbContext)
         public BasicApiContext DbContext { get; }
 
         [HttpGet("{id}", Name = "FindPetById")]
+        [ProducesResponseType(typeof(Pet),  StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
```

Comment:
@pranavkm these additions quiet the analyzer down. But, please verify they're sensible.

Reply:
There's an issue to move the analyzers to a separate assembly - https://github.com/aspnet/Mvc/issues/8153. I'm going to try and get that in today. That should avoid the need to add these.

Reply:
Any reason _not_ to add the attributes? I originally did it to avoid build failures but am not seeing the harm.